src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -228,6 +228,8 @@ case class WorkspacePermissionsPair(workspaceId: String,
 
 case class WorkspaceStatus(workspaceName: WorkspaceName, statuses: Map[String, String])
 
+case class BucketUsageResponse(usageInBytes: BigInt)
```

Comment:
i personally don't object to this but i'll defer to @dvoet 

we have a bunch of model classes so we'd want to change them there too